vuepress/docs/next/docs/getting-started/landing-page.md

```diff
@@ -39,6 +40,12 @@ The [Entando Standard Banking Demo](../../tutorials/solution/install-standard-de
 
 <img src="./landing-images/standard_demo2.png" width="533" height="250">
 
+## Entando Strapi
+
+The [Entando Strapi](../../tutorials/solution/strapi.md) packaged business capability (PBC) extends the versatility, flexibility and scope of the App Builder CMS by integrating Strapi into an Entando Application. The configuration, template and content widgets demonstrate Entando's ability to interface with best-of-breed platforms to provide the user with a variety of standard or cutting-edge tools and resources. In this way, Entando facilitates development instead of imposing limitations.
```

Comment:
not a big issue, but it might be cleaner to end the second sentence earlier >> The configuration, template and content widgets demonstrate Entando's ability to interface with best-of-breed platforms 
or 
The configuration, template and content widgets demonstrate Entando's ability to interface with best-of-breed platforms, providing the user with a variety of tools. 

the last bit of that sentence seems kind of circular with best-of-breed >> cutting-edge, I know they are different but simplifying that bit leads into the last sentence more smoothly, to end on that punch about not imposing limitations

Reply:
we can omit "standard or cutting-edge" - just wanted to highlight that entando is compatible with what devs love using and also the new stuff hitting the industry